bootstrap/src/routes/Welcome.js

```diff
@@ -161,6 +161,32 @@ const Welcome = () => {
                 .
             </p>
 
+            <h3>Add multi-tenant functionality and optional project picker</h3>
+            <ul>
+                <li>
+                    In <Code>Home.js</Code> replace all hard-coded project ID's with the one provided by
+                    <Code>useProjectId</Code> hook:{" "}
+                    <Code> {`<Headline projectId={projectId} … > … </Headline>`}</Code>
+                </li>
+                <li>
+                    <Code>ProjectId</Code> context object in <Code>/src/contexts/ProjectId.js</Code> stores
```

Comment:
The <Code>ProjectId</Code> context object in <Code>/src/contexts/ProjectId.js</Code> stores